source/references.adoc

```diff
@@ -1,19 +1,28 @@
 [appendix]
 = References
 
-Below are references to various topics of interest around RPMs, RPM Packaging,
-and RPM Building. Some of these will be advanced and extend far beyond the
-introductory material included in this guide.
+Below are references to various topics of interest around RPMs, RPM packaging, and RPM building. Some of these are advanced and extend far beyond the introductory material included in this guide.
+
+ifdef::rhel[]
+
+link:https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections Overview] - The Red Hat Software Collections offering provides continuously updated development tools in latest stable versions.
+
+link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html/packaging_guide/ [Red Hat Software Collections] - The Packaging Guide provides an explanation of Software Collections and details how to build and package them.  Developers and system administrators with basic understanding of software packaging with RPM can use this Guide to get started with Software Collections.
```

Comment:
The extra space between `packaging_guide/` and `[Red` breaks formatting, I think.